EDU-6407 fix: updated secure journey to api v4 (#1809)
aziontech/docs: src/content/docs/en/pages/secure-journey/transport-layer-security/ciphers.mdx

---

import Tabs from '~/components/tabs/Tabs'
import Code from '~/components/Code/Code.astro'
import Apiv4Rollout from '~/includes/snippets/apiv4Rollout/en/snippet.mdx'

<Apiv4Rollout />

HTTPS applications require additional security configurations in the form of TLS cryptography. When you [configure an edge application](/en/documentation/products/guides/build/configure-main-settings/) with Azion, you can select the minimum TLS version supported and which cipher suite is used by the application.

> There are separate instructions for API v3, which uses Delivery settings previously located within Edge Application Main Settings, and for API v4, where Delivery Settings have been moved to the new [Workloads](/en/documentation/products/secure/workloads) product.

Reviewer: ```suggestion
There are separate instructions for API v3, which uses **Delivery** settings previously located within **Edge Application **Main Settings. On API v4, **Delivery** settings have been moved to the new [Workloads](/en/documentation/products/secure/workloads) product.
```

:::tip
In this guide, there are separate instructions for API v3 and API v4. If you're not sure which steps apply to your account, see [the Verify Your Account Migration guide](/en/documentation/products/guides/verify-account-migration/) to determine if your account has already been migrated.
:::

---

## Configuring an HTTPS application

To enable the HTTPS protocol for your application:

<Tabs client:visible>
    <Fragment slot="tab.consoleworkloads">Console - Workloads</Fragment>
    <Fragment slot="tab.consoleedgeapplication">Console - Edge Application</Fragment>
    <Fragment slot="tab.apiv3">API v3</Fragment>
    <Fragment slot="tab.apiv4">API v4</Fragment>

<Fragment slot="panel.consoleworkloads">

1. Access [Azion Console](/en/documentation/products/guides/how-to-access-azion-console/) > **Workloads**.
2. Click the Workload you want to configure.
3. In **Protocol Settings**, select the **HTTP and HTTPS support**.

> Now to specify the [minimum TLS version](/en/documentation/products/build/edge-application/main-settings/#minimum-tls-version) and the [cipher suite](/en/documentation/products/build/edge-application/main-settings/#tls-ciphers) supported by your application:

Reviewer: ```suggestion
To specify the [minimum TLS version](/en/documentation/products/build/edge-application/main-settings/#minimum-tls-version) and the [cipher suite](/en/documentation/products/build/edge-application/main-settings/#tls-ciphers) supported by your application, follow these steps:
```
